portfolio/src/main/java/com/google/sps/data/Cafe.java

```diff
@@ -15,12 +15,16 @@
 package com.google.sps.data;
 
 /* Represents a user-uploaded coffee shop at specific address */
-public class Cafe {
-  String address;
-  boolean isStarbucks;
+public final class Cafe {
+  private final String address;
+  private final boolean isStarbucks;
 
-  public Cafe(String address, boolean starbucks) {
+  private Cafe(String address, boolean starbucks) {
     this.address = address;
     this.isStarbucks = starbucks;
   }
+
+  public static Cafe makeNewCafe(String address, boolean starbucks) {
```

Comment:
This doesn't really have much benefit in this case. We make a separate static creator when there is a complex business logic to call the constructor. There is no reason to hide the constructor if the parameters are identical.

Reply:
Done!